lib/output/cli-output.js

```diff
@@ -53,7 +53,7 @@ function getBodyContent(results, flags) {
     chalk`{underline {bold {white Resolved}}}`,
     chalk`{underline {bold {white Latest}}}`,
     chalk`{underline {bold {white Violation Type}}}`,
-    chalk`{underline {bold {white Deprecation Date}}}`,
+    chalk`{underline {bold {white End of Support}}}`,
```

Comment:
non-blocking: I would love your opinion between `Support Ends` & `End of Support`

Reply:
I feel `Support Ends` talks about the future dates, rather than past dates more often. `End of Support` is more generic, it can be the past date or future date.